app/kubernetes/converters/deployment.js

```diff
@@ -22,7 +22,9 @@ class KubernetesDeploymentConverter {
     const res = new KubernetesDeployment();
     res.Namespace = formValues.ResourcePool.Namespace.Name;
     res.Name = formValues.Name;
-    res.StackName = formValues.StackName ? formValues.StackName : formValues.Name;
+    if (formValues.StackName) {
+      res.StackName = formValues.StackName;
```

Comment:
Does portainer handle the case where the `StackName ` field is omitted ok?

Reply:
Yes